src/hidimstat/base_perturbation.py

```diff
@@ -241,18 +176,20 @@ def _joblib_predict_one_group(self, X, group_id, group_key):
         ----------
         X: array-like of shape (n_samples, n_features)
             The input samples.
-        group_id: int
+        feature_group_id: int
             The index of the group of variables.
-        group_key: str, int
+        feature_group_key: str, int
             The key of the group of variables. (parameter use for debugging)
```

Comment:
We should probably not keep this parameter ?

Reply:
Why do you want to remove it?

For me, it's an internal method and it's essential for the debugging.

Reply:
@bthirion

Reply:
This case is quite particular because it's run in parallel and the thread or the process doesn't have the mapping between the feature_group_key and the feature_group_id. Without this mapping it will be difficult to debug the function because most of the time, we will need to look at one specific feature and it's easier to use the name of it instead a random mapping.

Reply:
see issue #472